Sofya_Huliuk_Application/Program.cs

```diff
@@ -294,8 +294,161 @@ public void Task_9()
         uint d = (uint)b;
     }
 
+    public void Task_10()
+    {
+
+    }
+
 
 
 
 
 }
+
+class Homework_3
+{
+    public void Task_1()
+    {
+        while (true)
+        {
+            Console.WriteLine("Enter the integer:");
+            string input = Console.ReadLine();
+            bool result = int.TryParse(input, out var num);
+            if (!result)
+
+            {
+                Console.WriteLine("incorrect value, please try again");
+                continue;
+            }
+
+            if (num % 10 == 0)
```

Comment:
The condition should be "if is divisible by both 2 and 5"

Reply:
"if it is divisible by both 2 and 5" then it is divisible by 10 too, these are the same things